packages/documentation/src/stories/components/input/input.snapshot.stories.ts

```diff
@@ -0,0 +1,112 @@
+import type { Args, StoryContext, StoryObj } from '@storybook/web-components';
+import meta from './input.stories';
+import { html } from 'lit';
+import { getCombinations, COMBINATIONS } from '../../../utils/inputComponentsGetCombinations';
+
+export default {
+  ...meta,
+  title: 'Snapshots',
+  render: renderInputSnapshot,
+};
+
+function renderInputSnapshot(_args: Args, context: StoryContext) {
+  const combinations = [
+    ...COMBINATIONS,
+    {
+      label: `Label - no Placeholder`,
+      placeholder: null,
+    },
+    {
+      label: `Label - with Value`,
+      value: 'Lorem Ipsum',
+    },
+  ];
+  return html`
+    <div class="d-flex flex-wrap align-items-start gap-regular">
+      ${['bg-white', 'bg-dark'].map(
+        bg => html`
+          <div class="${bg} d-flex gap-3 flex-column p-3">
+            <h3>Sizes</h3>
+            ${getCombinations('size', context.argTypes.size.options, combinations)
+              .filter(
+                (args: Args) =>
+                  !args.value ||
+                  (args.value &&
+                    (context.args.type === 'text' || context.args.type === 'password')),
+              )
+              .map(
+                (args: Args) =>
+                  html`
+                    <div>
+                      ${args.title !== undefined && args.title
+                        ? html`
+                            <h4>${args.size}</h4>
```

Comment:
I would like to get the control.label value, instead of the key.
Not sure if this works.

```suggestion
                            <h4>${Object.entries(context.argTypes.size.control.labels).find(([key, value]) => key === args.size)[1]}</h4>
```